CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,88 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
+- Support DSR mode for Service's external addresses in AntreaProxy, including LoadBalancerIPs and ExternalIPs. ([#5202](https://github.com/antrea-io/antrea/pull/5202) [#5251](https://github.com/antrea-io/antrea/pull/5251), [@tnqn])
+- Containerize Windows userspace OVS processes and run them in a container of the Antrea Agent Pod to align with the Linux design. ([#4936](https://github.com/antrea-io/antrea/pull/4936) [#5052](https://github.com/antrea-io/antrea/pull/5052), [@rajnkamr] [@Atish-iaf])
```

Comment:
link to PR #5305  could be added as well here?

Reply:
Added.